src/chain.h

```diff
@@ -24,6 +24,7 @@ static const int SAPLING_VALUE_VERSION = 1010100;
 static const int CHAIN_HISTORY_ROOT_VERSION = 2010200;
 static const int NU5_DATA_VERSION = 4050000;
 static const int TRANSPARENT_VALUE_VERSION = 5040026;
+static const int NU6_DATA_VERSION = 6000000; //FIXME, how do we assign these?
```

Comment:
```suggestion
static const int NU6_DATA_VERSION = 5100025;
```
This means zcashd v5.10.0-rc1 which will (probably) be the first release containing this change.